Src/IronPython/Runtime/SimpleNamespace.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using IronPython.Runtime.Operations;
+using IronPython.Runtime.Types;
+using Microsoft.Scripting;
+using Microsoft.Scripting.Runtime;
+using NotNullAttribute = Microsoft.Scripting.Runtime.NotNullAttribute;
+
+namespace IronPython.Runtime {
+    [PythonHidden, PythonType("types.SimpleNamespace")]
+    public class SimpleNamespace {
+        public SimpleNamespace([ParamDictionary, NotNull]Dictionary<string, object?> kwargs\u00F8) {
```

Comment:
Why "kwargsø" and not "kwargs"?

Reply:
It's a trick to avoid having the kwargs keyword try to bind to the argument. For example, to allow `types.SimpleNamespace(kwargs=1)`. It's used in a few places in the codebase (for example if you search for uses of `ParamDictionaryAttribute`, of `StringOps.format`). Ideally the binding should be prevented with some sort of attribute instead...

Reply:
OK, thanks for the explanation!